src/azure-cli/azure/cli/command_modules/resource/tests/hybrid_2019_03_01/test_managmentgroups.py

```diff
@@ -155,25 +169,28 @@ def test_create_managementgroup(self):
         self.cmd('account management-group delete --name ' + name)
 
         self.assertIsNotNone(managementgroup_create)
-        self.assertIsNotNone(managementgroup_create["properties"]["details"])
+        self.assertIsNotNone(managementgroup_create["details"])
         self.assertEqual(
             managementgroup_create["id"],
             "/providers/Microsoft.Management/managementGroups/" + name)
         self.assertEqual(managementgroup_create["name"], name)
         self.assertEqual(
-            managementgroup_create["properties"]["displayName"],
+            managementgroup_create["displayName"],
             displayName)
         self.assertEqual(
-            managementgroup_create["properties"]["details"]["parent"]["id"],
+            managementgroup_create["details"]["parent"]["displayName"],
+            "Root Management Group")
+        self.assertEqual(
+            managementgroup_create["details"]["parent"]["id"],
             "/providers/Microsoft.Management/managementGroups/" +
-            managementgroup_create["properties"]["tenantId"])
+            managementgroup_create["tenantId"])
         self.assertEqual(
-            managementgroup_create["properties"]["details"]["parent"]["name"],
-            managementgroup_create["properties"]["tenantId"])
-        self.assertIsNotNone(managementgroup_create["properties"]["tenantId"])
```

Comment:
Why delete `["properties"]`?

Reply:
The json formatting changed when updating the API version, removing `[properties]`